edxadapt/edxadapt.py

```diff
@@ -60,7 +60,27 @@ class EdxAdaptXBlock(StudioEditableXBlockMixin, XBlock):
         scope=Scope.preferences
     )
 
-    editable_fields = ('params', 'skills', 'edx_adapt_api_url')
+    success_registration_msg = String(
+        default=_(
+            "Successfully registered in Edx Adapt, course is completely ready to start."
```

Comment:
"Your adaptive learning session is configured now. Please click Next button to start solve problems."